src/store/personalSettings/index.ts

```diff
@@ -31,37 +31,21 @@ const mod = {
 
       commit.SET_PERSONAL_SETTING(response);
     },
-    async saveAditionalInfo(
+    async saveUserProfile(
       context: ActionContext<PersonalSettingsState, RootState>,
-      message: string
+      message: PersonalSetting
```

Comment:
I'd change the name of this variable from `message` to `personalSettings`

Reply:
Done. Renamed to setting